libs/payments/requests.go

```diff
@@ -19,15 +19,60 @@ type SubmitRequest struct {
 	PayoutID   string `json:"payoutId" valid:"required"`
 }
 
-// AddressApprovalRequest is provided to indicate approval of an on-chain address.
-type AddressApprovalRequest struct {
-	Address string `json:"address" valid:"required"`
-}
-
 // SubmitResponse is returned to provide the status of a payment after submission, along with any
 // error that resulted, if necessary.
 type SubmitResponse struct {
 	Status              PaymentStatus `json:"status" valid:"required"`
 	PaymentDetails      `json:"paymentDetails,omitempty"`
 	ExternalIdempotency string `json:"externalIdempotency,omitempty"`
 }
+
+// AddressApprovalRequest is provided to indicate approval of an on-chain address.
+type AddressApprovalRequest struct {
+	Address string `json:"address" valid:"required"`
+}
+
+// OperatorShare represents the association between an operator name and their encrypted share
+type OperatorDataResponse struct {
+	Name     string `json:"name" valid:"required"`
+	Material []byte `json:"material" valid:"required"`
+}
+
+type OperatorDataRequest struct {
```

Comment:
To me it seems like this operator data should be explicitly linked to the authorizer keys: https://github.com/brave-intl/bat-go/blob/nitro-payments-dev/services/payments/authorize.go#L16-L35

I'm thinking the best answer here is to pull this operator data directly from a static hardcoded file included in source so that the image data being attested to by the enclave encapsulates who the operators are that can authorize the actions.

I can't think of a case where we'd want an operator to not also be an authorizer or vice versa.